config/local.yaml

```diff
@@ -360,7 +360,7 @@ checks:
     options:
       postage-amount: 1000
       postage-depth: 17
-      postage-topup-amount: 100
+      postage-topup-amount: 1100
```

Comment:
let's go back up 100, this should also works. If it doesn't work, we have to understand why.